packages/vscode/README.md

```diff
@@ -28,9 +30,10 @@ You can find both the [stable](https://open-vsx.org/extension/Prisma/prisma) and
 - Documentation help
   - Documentation of a completion result pops up as completion results are provided.
 - Quick info on hover
-  - Documentation Comments (`///`) of models and enums appear anywhere you hover over their usages.
+  - Documentation Comments (`///`) of `model`, `enum` or `type` appear anywhere you hover over their usages.
+  - CMD + hover on a field whose type is a `model`, `enum` or `type` will show the corresponding block in a popup.
 - Go to Definition
-  - Jump to or peek a model or enum's declaration.
+  - Jump to or peek a `model`, `enum` or `type` definition. (CMD + left click)
```

Comment:
Does a readme file maybe support `<kbd>` to make <kbd>CMD</kbd> maybe?

Reply:
Nice
```suggestion
  - Jump to or peek a `model`, `enum` or `type` definition. (<kbd>CMD</kbd> + left click)
```